packages/fxa-auth-server/lib/routes/totp.js

```diff
@@ -599,15 +601,23 @@ module.exports = (
           await request.emitMetricsEvent('totpToken.verified', { uid });
           // this signals the end of the login flow
           await request.emitMetricsEvent('account.confirmed', { uid });
-          glean.login.totpSuccess(request, { uid });
+
+          if (!isSetup) {
+            // Emit a login success event
+            glean.login.success(request, { uid });
```

Comment:
Noticed we have login.complete (emitted via flow complete signal) and login.success events - are these different?